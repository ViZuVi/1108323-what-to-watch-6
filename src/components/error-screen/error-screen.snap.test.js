import React from 'react';
import {render} from '@testing-library/react';
import ErrorScreen from './error-screen';

test(`ErrorScreen should render correctly`, () => {
  const {container} = render(
      <ErrorScreen error={``} />);
  expect(container).toMatchSnapshot();
});

