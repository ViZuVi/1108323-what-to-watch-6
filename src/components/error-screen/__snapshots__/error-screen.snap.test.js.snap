// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorScreen should render correctly 1`] = `
<div>
  <div>
    <p
      style="text-transform: uppercase; font-size: 32px; text-align: center; color: black;"
    >
      Error
    </p>
    <p
      style="font-size: 24px; text-align: center; color: black;"
    >
      Try again later
    </p>
  </div>
</div>
`;
